spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/db/migration/mariadb/V11__Remove_Task2_Batch4_Support.java

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.server.db.migration.mariadb;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.springframework.cloud.dataflow.common.flyway.SqlCommand;
+import org.springframework.cloud.dataflow.server.db.migration.AbstractRemoveBatch4Task2Tables;
+
+/**
+ * Prefixes Task V2 tables and V4 Batch tables with a V2_prefix as well as remove the BOOT3_ prefix for V3 task and v5 batch tables.
+ *
+ * @author Glenn Renfro
+ */
+public class V11__Remove_Task2_Batch4_Support extends AbstractRemoveBatch4Task2Tables {
+
+	/**
+	 * Scripts to remove views used for Task V2/Batch V4 Task V3/Batch V5 queries.
+	 */
+	private final static String DROP_VIEW_AGGREGATE_TASK_EXECUTION =
+		"DROP VIEW AGGREGATE_TASK_EXECUTION";
+
+	private final static String DROP_VIEW_AGGREGATE_TASK_EXECUTION_PARAMS =
+		"DROP VIEW AGGREGATE_TASK_EXECUTION_PARAMS";
+
+	private final static String DROP_VIEW_AGGREGATE_JOB_EXECUTION =
+		"DROP VIEW AGGREGATE_JOB_EXECUTION";
+
+	private final static String DROP_VIEW_AGGREGATE_JOB_INSTANCE =
+		"DROP VIEW AGGREGATE_JOB_INSTANCE";
+
+	private final static String DROP_VIEW_AGGREGATE_TASK_BATCH =
+		"DROP VIEW AGGREGATE_TASK_BATCH";
+
+	private final static String DROP_VIEW_AGGREGATE_STEP_EXECUTION =
+		"DROP VIEW AGGREGATE_STEP_EXECUTION";
+
+	/**
```

Comment:
**[GlobalComment]** I think these should just be regular comments rather than javadoc, otherwise they will be attributed to the variable immediately below it, but nothing else.